### PR TITLE
New home people news style

### PR DESCRIPTION
**Pull request by SaharMehrpour**
New changes:

- removed `minima` theme from `_config.yml`
- added bootstrap in `head`
- added css styles in `assets/css` and imported them in `head`

- updated the style of `home` and `people` pages with bootstrap
- moved pictures to the `gallery` page to match Abdulaziz design
- edited and renamed a few files

- added a `News` page -- Only the 3 recent news truncated to 150 chars are shown in `home page, all news but truncated to 150 chars are shown in all news page. All the content of the news is available on its own page (not sure if it is a good design!)

### CODE REVIEW

**Comment by luminaxster** [comment]
Awesome! I love all the visual changes. However, we should use the _layouts for styling only and read the content from the MD files. We could enhance minima rather than getting rid of it since the navigation bar is now hardcoded.

File: docs_src/_config.yml [hunk 1]
```diff
@@ -54,12 +54,12 @@ scholar:
   query: "@*"
 
 # Build settings
-theme: minima
-header_pages:
-  - projects.markdown
-  - publications.markdown
-  - people.html
-  - about.markdown
+#theme: minima
```
Comment (luminaxster) [inline docs_src/_config.yml]: we could keep these built-in navigation

File: docs_src/_includes/header.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <a href="{{ site.baseurl }}/"><img class="banner-img" src="{{site.baseurl}}/assets/img/logo.png"></a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav"
+                aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav ml-auto">
+
+                {% assign navButtons = "home|projects|publications|people|gallery|about" | split: "|" %}
```
Comment (luminaxster) [inline docs_src/_includes/header.html]: not fan of hard-coded navigation.

File: docs_src/_layouts/home.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+    <div class="summary row">
+        <div class="col-lg-6 pt-4 pt-lg-0 order-2 order-lg-1 d-flex flex-column justify-content-center footer-top">
+            <h1>We are DevX!</h1>
+            <h2>We study how humans interact with code and design new ways to build software. I work at the intersection
+                of software engineering and human-computer interaction.</h2>
+        </div>
+        <div class="col-lg-6 order-1 order-lg-2 hero-img">
+            <img src="{{ site.baseurl }}/assets/img/gallery/oh-george-summer19.jpg" class="img-fluid" alt="Group Photo">
+        </div>
```
Comment (luminaxster) [inline docs_src/_layouts/home.html]: this could be obtained from the md file

File: docs_src/_layouts/home.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        reuse, design, test, and understand code. Experienced developers work better by using more effective
+        strategies. We've been exploring ways to capture and make explicit the strategies developers use, helping
+        uncover how developers work differently and to create new ways to help developers benefit from the
+        hard-earned wisdom of experienced developers.
+    </p>
+
+    <h3>Debugging Hypotheses</h3>
+    <p>A sample .bib reference:</p>
+    {% bibliography -q @*10.1109/CHASE.2019.00035 %}
+
```
Comment (luminaxster) [inline docs_src/_layouts/home.html]: this should be injected from the md file and formatted here

File: docs_src/gallery.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+            <div class="gallery-item">
+                <a href="{{site.baseurl}}/assets/img/gallery/patriothack.jpg" class="venobox vbox-item" data-gall="gallery-item">
+                    <img src="{{site.baseurl}}/assets/img/gallery/patriothack.jpg" alt="" class="img-fluid">
+                </a>
+            </div>
+        </div>
+
+    </div>
+
+</div>
```
Comment (luminaxster) [inline docs_src/gallery.html]: this should be similar to news.html instead of harcoded.

**Review by luminaxster — COMMENTED** [review]
this was a quick review, I think that we can build on top of minima: https://github.com/jekyll/minima#customizing-templates, like here:  https://github.com/sparkrnyc/sparkr_general_docs/wiki/How-to-Add-Bootstrap-to-Jekyll

File: docs_src/_includes/header.html (hunk 2 above)
Comment (SaharMehrpour) [reply docs_src/_includes/header.html]: I agree. I assumed the navigation bar is not gonna change frequently, so it is fine to be hard-coded. :)

File: docs_src/_layouts/home.html (hunk 4 above)
Comment (SaharMehrpour) [reply docs_src/_layouts/home.html]: I haven't touched this part yet. I agree it should be read from `md` files.

File: docs_src/_layouts/home.html (hunk 3 above)
Comment (SaharMehrpour) [reply docs_src/_layouts/home.html]: Only the image or the whole content? How?

File: docs_src/_includes/header.html (hunk 2 above)
Comment (SaharMehrpour) [reply docs_src/_includes/header.html]: I think if we bring back the Minima theme, we can remove this file.

**Comment by luminaxster** [comment]
we can now do PRs to the main repo rather than my fork. I answered your other question: https://github.com/devuxd/devx.cs.gmu.edu/issues/4

**Pull request closed without merging**